agreement_legal/static/src/js/domain_widget_ext.js

```diff
@@ -39,8 +39,10 @@ odoo.define("agreement_legal.domain_widget_ext", function(require) {
 
             // Create the domain selector or change the value of the current
             // one...
-            /* eslint-disable no-negated-condition */
-            if (!this.domainSelector) {
+            var def = null;
```

Comment:
```suggestion
            let def = null;
```

Reply:
Thanks @Tardo but in the meantime it was already merged... I guess this is not too important, so I will take it into account next time!